functional_tests/imports/01-batch1.py

```diff
@@ -8,7 +8,9 @@
 plugin:
   - wandb
 depend:
-  pip_install_timeout: 1500  # 25m
+  pip_install_timeout: 3000  # 50m
```

Comment:
this PR should fix your issue with imports test:
https://github.com/wandb/client/pull/3167

Reply:
thank you!